HiP-DataStore.Model/Rest/ReviewArgs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Rest
+{
+    public class ReviewArgs
+    {
+
+        public string Description { get; set; }
+
+        public List<string> Reviewers { get; set; }
```

Comment:
It should be made clear that a list of Id's has to be passed here and not a list of usernames. A comment should be enough